docs/segments/manage_segments.rst

```diff
@@ -3,4 +3,493 @@
+
+.. vale on
+
+You can create dynamic Segments by using date filters.
+
+.. note:: 
+  
+  The date format for values stored in the database is ``YYYY-MM-DD``. For example, December 11, 2020 is stored as 2020-12-11 and November 12, 2020 is stored as 2020-11-12. To update the display format for dates, go to **Settings > Configuration > System Settings > System Defaults**. However, this doesn’t alter the format in which dates are stored in the database.
+
+.. vale off
+
+Operators
+=========
+
+.. image:: images/operators.png
+    :alt: Screenshot showing Operators.
+
+.. vale on
+
+You must ensure that you use the correct operator and time frame to build an effective Segment.
+
+Once you’ve selected a date field as your filter, such as the default **Date last active** field or a custom **Birthday** field, you’ll have a list of operators to choose from:
+
+* **Equals** - The value on the Contact record exactly matches the filter value.
+
+* **Not Equal** - The value on the Contact record is any value that doesn’t match the filter value.
+
+* **Greater than** - The value on the Contact record is at a later date in time than X date. For example, **Greater than** today means anytime from tomorrow until the end of time.
+
+* **Greater than or equal** - The value on the Contact record is either at a later date in time than or including X date. For example, **Greater than or equal** today means anytime from today until the end of time
+
+* **Less than** - The value on the Contact record is at an earlier date in time than X date. For example, **Less than today** means anytime from the beginning of time until yesterday.
+
+* **Less than or equal** - The value on the Contact record is at an earlier date in time than X date. For example, **Less than or equal today** means anytime from the beginning of time until today.
+
+* **Empty** - No value exists in the field on the Contact record.
+
+* **Not empty** - A value exists in the field on the Contact record.
+
+* **Like** - This operator isn't supported in date or date-time fields, and shouldn’t be used.
+
+* **Not like** - This operator isn't supported in date or date-time fields, and shouldn’t be used.
+
+* **Regexp** - Segment includes Contacts with values that match the specified regular expression pattern. If you aren't proficient with regular expression, don’t use this operator.
+
+* **Not regexp** - Segment includes Contacts with values that don’t match the specified regular expression pattern. If you aren't proficient with regular expressions, don’t use this operator.
+
+* **Starts with** - Segment includes Contacts whose field values begin with the specified numbers. These filter values should generally reference years (or years and months).
+
+For example, A value of 19 matches any Contacts whose field value has a year in the 1900&nbsp;s. A value of 200 matches Contacts with a year value between 2000 and 2009 and a value of 2020-11 matches Contacts with a field value in November 2020.
```

Comment:
This looks a bit out of place as it has escaped the list. Please also fix the ascii characters.